x-pack/plugins/observability/public/components/threshold/threshold_rule_expression.tsx

```diff
@@ -358,7 +377,7 @@ export default function Expressions(props: Props) {
   }
 
   const placeHolder = i18n.translate(
-    'xpack.observability.threshold.rule.homePage.toolbar.kqlSearchFieldPlaceholder',
+    'xpack.observability.threshold.rule.alertFlyout.searchBar.placeholder',
     {
       defaultMessage: 'Search for infrastructure data… (e.g. host.name:host-1)',
```

Comment:
should we change the message to something like `'Search for observability data… (e.g. host.name:host-1)'`

Reply:
Sure, I'll change it